kernel_gateway/gatewayapp.py

```diff
@@ -312,19 +320,23 @@ def init_configurables(self):
                     self.prespawn_count, self.max_kernels)
                 )
 
-        if self.api == 'notebook-http':
-            self.kernel_pool = ManagedKernelPool(
-                self.prespawn_count,
-                self.kernel_manager
-            )
-        else:
-            self.kernel_pool = KernelPool(
-                self.prespawn_count,
-                self.kernel_manager
-            )
+        
+        api_module = self._load_api_module(self.api)
+        func = getattr(api_module, 'create_personality', None)
```

Comment:
That's what happens when I do code review at night. Somehow I missed the `getattr` and read it as invoking `creating_personality`. 👀 

Yes. I'd let the AttributeError bubble. If they've passed a valid package that doesn't have that function, let's fail fast and ugly.